ors-api/src/test/java/org/heigit/ors/api/services/APIServiceTest.java

```diff
@@ -19,56 +21,72 @@
 
 import static org.junit.jupiter.api.Assertions.*;
 
-class APIRequestTest {
-    APIRequest request;
+class APIServiceTest {
+    AbstractApiService apiService;
 
+    static class ApiService extends AbstractApiService {
```

Comment:
Might become obsolete once the abstract methods in `AbstractApiService` become default implementations as suggested in my other comments.